cmd/exo/stop.go

```diff
@@ -7,8 +7,13 @@ import (
 	"github.com/spf13/cobra"
 )
 
+var initTimeout uint = 0
```

Comment:
you don't need this `initTimeout` variable, since it is only used in the `.Flags()` call. That line can instead do `&timeoutSeconds` if `var timeout int` instead of `*int`.

Reply:
Ah but I need to pass a pointer to `ws.Stop` that is possibly `nil`. If `timeoutSeconds` is `uint` rather than `*uint` I can't do that, since you can't do `&timeoutSeconds = nil`.